docs/s3-supported-api.md

```diff
@@ -148,6 +149,9 @@
 10. Delete Object tagging  
     `aws s3api delete-object-tagging --bucket <your_bucket> --key <key>`  
 
+11. Copy Object  
+    `aws s3api copy-object --copy-source bucket-1/test.txt --key test.txt --bucket bucket-2`
```

Comment:
For object names, we can have formatted value, like :
`aws s3api copy-object --copy-source <source_key> --key <dest_key> --bucket <dest_bucket>`

Reply:
done